### PR TITLE
543 create project structure

### PR DESCRIPTION
**Pull request by Bhupesh-mfsi**
Comment : Created project structure
Task details : https://github.com/corpus-io/dm3/issues/543

### CODE REVIEW

File: packages/messenger-demo/src/Background.ts
Comment (hai-ko) [inline packages/messenger-demo/src/Background.ts:1]: This file can be removed

File: packages/messenger-demo/package.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        "@types/react": "^18.0.13",
+        "@types/react-dom": "^18.0.5",
+        "autoprefixer": "10.4.5",
+        "bootstrap": "^5.1.3",
+        "messenger-widget": "0.2.0",
+        "postcss": "^8.4.24",
+        "react": "^18.2.0",
+        "react-dom": "^18.2.0",
+        "react-scripts": "5.0.1",
+        "three": "^0.142.0",
```
Comment (hai-ko) [inline packages/messenger-demo/package.json:20]: This can be removed

**Review by hai-ko — CHANGES_REQUESTED** [review]

File: packages/messenger-demo/src/Background.ts
Comment (Bhupesh-mfsi) [reply packages/messenger-demo/src/Background.ts:1]: Removed

File: packages/messenger-demo/package.json (hunk 1 above)
Comment (Bhupesh-mfsi) [reply packages/messenger-demo/package.json]: Removed

**Review by hai-ko — APPROVED** [review]

**Pull request merged**